hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManager.java

```diff
@@ -80,4 +90,40 @@ public void createConsumerGroup(Topic topic, Subscription subscription) {
           e);
     }
   }
+
+  @Override
+  public void deleteConsumerGroup(SubscriptionName subscriptionName)
```

Comment:
can we log datacenter instead of / additionally to cluster? Unfortunately cluster name does not give full context in multi dc setup

Reply:
The clusterName property contains the value in the format `{clustername}-{datacenter}`. 
I’ve renamed it to qualifiedClusterName, but if you think it would be clearer to separate them, I can adjust it accordingly.